api.js

```diff
@@ -18,4 +18,11 @@ const getArticleById = (article_id) => {
   });
 };
 
-export { getArticles, getArticleById };
+const getCommentsByArticleId = (article_id) => {
+  const url = `/articles/${article_id}/comments`;
+  return api.get(url).then(({ data }) => {
+    return data.comments;
+  });
+};
```

Comment:
Nice, but url isnt the most appropriate name for this, I would recommend using something along the lines of endpoint instead. This is mainly because url itself I would assume is a full url, but this is just a specific endpoint we are wanting to make a request to.